src/components/verification-grid-tile/css/style.css

```diff
@@ -75,3 +75,56 @@
   background-color: var(--oe-panel-color);
   border-radius: var(--oe-border-rounding);
 }
+
+.progress-meter {
+  --progress-meter-rounding: var(--oe-border-rounding);
+
+  position: relative;
+  display: flex;
+  height: var(--oe-spacing);
```

Comment:
that's an interesting choice.

I would have assumed line height would be appropriate so it would scale with font sizes.